packages/bot-web-ui/src/stores/load-modal-store.ts

```diff
@@ -458,17 +458,20 @@ export default class LoadModalStore implements ILoadModalStore {
             const load_options = { block_string: e.target.result, drop_event, from: save_types.LOCAL };
             if (is_preview) {
```

Comment:
```suggestion
            if (is_preview && ref) {
```

can we reduce the `if` statement to 1 to reduce the cognitive complexity of this code here ?

Reply:
Alright @amir-deriv, But do note however that with this change, we alter the logic a bit, as previously, the `else` block was only executed if `is_preview` was `false`,  now it'll be executed if `is_preview` is `true`, and `ref` is `null` or `undefined`. Unless you're also suggesting I turn the `else` block into a `else if` 🤔